src/modules/account/components/MaterialRequestDetailBlade/MaterialRequestDetailBlade.tsx

```diff
@@ -45,11 +45,11 @@ const MaterialRequestDetailBlade: FC<MaterialRequestDetailBladeProps> = ({
 		<Blade
 			isOpen={isOpen}
 			renderTitle={() => (
-				<h2 className={styles['p-account-my-material-requests__title']}>
+				<h4 className={styles['p-account-my-material-requests__title']}>
```

Comment:
Instead of tweaking the HTML outline, add a global classname that sets the same styling as an h4. This way, we don't ruin the outline of the page.

By default, the styling for an h4 has
```
	font-size: $font-size-lg;
	line-height: $line-height-lg;
```

In `Blade.module.scss`, there's already a classname added, you could add the same values there.